lib/sneakers/worker.rb

```diff
@@ -61,11 +61,11 @@ def do_work(delivery_info, metadata, msg, handler)
           end
         rescue Timeout::Error
           res = :timeout
-          worker_error('timeout')
+          worker_error('timeout',msg)
         rescue => ex
           res = :error
           error = ex
-          worker_error('unexpected error', ex)
+          worker_error('unexpected error', msg, ex)
```

Comment:
Perhaps we can add a config option , to Sneakers::Configuration? `error_log_message: <boolean>` and have `false` as default? But for my use case, I need to message to debug. I'm not sure how other people are debugging their apps, but I have many queues that flow through each other on various microservices.